scheduling/allocator.rb

```diff
@@ -180,7 +180,6 @@ def self.candidate_hosts(request)
       ds = ds.exclude(Sequel[:vm_host][:id] => request.host_exclusion_filter) unless request.host_exclusion_filter.empty?
       ds = ds.where(location: request.location_filter) unless request.location_filter.empty?
       ds = ds.where(allocation_state: request.allocation_state_filter) unless request.allocation_state_filter.empty?
-      ds = ds.exclude(total_cores: 14, total_cpus: 14) unless request.family == "standard-gpu"
```

Comment:
it might make sense to not put regular vms onto gex hosts. runners running on gex hosts get a very different performance characteristic. should we keep this commit?

Reply:
We are currently discussing how/if we'll leverage GEX going forward. We could keep this until the discussion has concluded.